typescript/src/start_script.ts

```diff
@@ -34,6 +34,8 @@ const argv = yargs
   .describe('output_dir', 'Path to a directory for the generated code')
   .alias('grpc-service-config', 'grpc_service_config')
   .describe('grpc-service-config', 'Path to gRPC service config JSON')
+  .alias('package-name', 'package-name')
```

Comment:
`.alias('package-name', 'package_name')` - note an underscore. The idea of those aliases is to allow both `--package-name` and `--package_name` because people tend to use both.

Reply:
I see, thanks for clarification!